### PR TITLE
fixed spacing issue on waypoint: counting cards

### LINKED ISSUE #6011: Add line break before "Example Output" for better readability
Challenge [Checkpoint: Counting Cards](http://www.freecodecamp.com/challenges/checkpoint-counting-cards#?solution=var%20count%20%3D%200%3B%0A%0Afunction%20cc(card)%20%7B%0A%20%20%2F%2F%20Only%20change%20code%20below%20this%20line%0A%20%20%0A%20%20%0A%20%20return%20%22Change%20Me%22%3B%0A%20%20%2F%2F%20Only%20change%20code%20above%20this%20line%0A%7D%0A%0A%2F%2F%20Add%2Fremove%20calls%20to%20test%20your%20function.%0A%2F%2F%20Note%3A%20Only%20the%20last%20will%20display%0Acc(2)%3B%20cc(3)%3B%20cc(7)%3B%20cc('K')%3B%20cc('A')%3B%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.106 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript
var count = 0;

function cc(card) {
  // Only change code below this line
  
  
  return "Change Me";
  // Only change code above this line
}

// Add/remove calls to test your function.
// Note: Only the last will display
cc(2); cc(3); cc(7); cc('K'); cc('A');

```

Right now the "Example Output" is smashed up against the previous paragraph:

image

Maybe give it some room like so:

image

### PR DESCRIPTION
**Pull request by AkiraLaine**
closes #6011

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/basic-javascript.json [hunk 1]
```diff
@@ -3978,12 +3977,7 @@
         "assert(lookUp(\"Akira\", \"address\") === \"No such property\", 'message: <code>\"Akira\", \"address\"</code> should return \"No such property\"');"
       ],
       "type": "checkpoint",
-      "challengeType": 1,
-      "nameCn": "",
-      "nameFr": "",
-      "nameRu": "",
-      "nameEs": "",
-      "namePt": ""
```
Comment (bugron) [inline seed/challenges/01-front-end-development-certification/basic-javascript.json]: Why did you removed these lines?

**Comment by AkiraLaine** [comment]
I might have done that by mistake. Shall I re-add them?

**Comment by bugron** [comment]
Yes, `amend` please.

**Comment by bugron** [comment]
OK, merged, thanks @AkiraLaine!

**Pull request merged**